guides/v2.1/frontend-dev-guide/css-topics/css-themes.md

```diff
@@ -6,7 +6,7 @@ functional_areas:
   - Theme
 ---
 
-## What's in this topic {#fedg_css-in-themes_overview}
+## In this topic {#fedg_css-in-themes_overview}
```

Comment:
Why do we need a heading like this?

Reply:
I was asking myself the same thing...then I saw a bunch of them.